test/NameRegistry.t.sol

```diff
@@ -257,30 +268,44 @@ contract NameRegistryTest is Test {
         assertEq(alice.balance, amount - nameRegistry.fee());
     }
 
-    function testRegisterWorksWhenAlreadyOwningAName(address alice, address recovery, bytes32 secret) public {
+    function testRegisterWorksWhenAlreadyOwningAName(
+        address alice,
+        address recovery,
+        bytes32 secret,
+        uint256 delay_bob,
+        uint256 delay_alice
+    ) public {
         _assumeClean(alice);
         _disableTrusted();
         vm.deal(alice, 1 ether);
         vm.warp(JAN1_2023_TS);
 
+        delay_alice = delay_alice % REASONABLE_SLA_PERIOD; // test delays up to REASONABLE_SLA_PERIOD && protect against overflow
+        delay_bob = delay_bob % REASONABLE_SLA_PERIOD; // test delays up to REASONABLE_SLA_PERIOD && protect against overflow
+        vm.assume(delay_alice > COMMIT_REPLAY_DELAY);
+        vm.assume(delay_bob > COMMIT_REPLAY_DELAY);
+
         // Register @alice to alice
         vm.startPrank(alice);
         bytes32 commitHashAlice = nameRegistry.generateCommit("alice", alice, secret, recovery);
         nameRegistry.makeCommit(commitHashAlice);
-        vm.warp(block.timestamp + COMMIT_REVEAL_DELAY);
+        vm.warp(block.timestamp + delay_alice);
         uint256 aliceRegister = block.timestamp;
         nameRegistry.register{value: nameRegistry.fee()}("alice", alice, secret, recovery);
 
+        // make this assertions before Alice's registration expires
+        assertEq(nameRegistry.ownerOf(ALICE_TOKEN_ID), alice);
+
         // Register @bob to alice
         bytes32 commitHashBob = nameRegistry.generateCommit("bob", alice, secret, recovery);
         nameRegistry.makeCommit(commitHashBob);
-        vm.warp(block.timestamp + COMMIT_REVEAL_DELAY);
+        vm.warp(block.timestamp + delay_bob);
         uint256 bobRegister = block.timestamp;
         nameRegistry.register{value: FEE}("bob", alice, secret, recovery);
         vm.stopPrank();
 
+        // remaining non-time sensitive assertions
```

Comment:
nit: can remove this the one on like 296 covers the important part

Reply:
would prefer to keep the commit on topic and not remove assertions. think removing extraneous assertions can be its own change.

Reply:
got it. will remove